src/feed/impl/TransactionFeed.cpp

```diff
@@ -178,6 +178,7 @@ TransactionFeed::pub(
         pubObj[JS(meta)] = rpc::toJson(*meta);
         rpc::insertDeliveredAmount(pubObj[JS(meta)].as_object(), tx, meta, txMeta.date);
         rpc::insertDeliverMaxAlias(pubObj[txKey].as_object(), version);
+        rpc::insertMPTIssuanceID(pubObj[JS(meta)].as_object(), tx, meta);
```

Comment:
Maybe take a ref to the `meta` object and reuse? Seems like we already have exactly the same above.

Reply:
could you elaborate? `meta` is already a shared pointer.

Reply:
Sorry i missed this comment and only seeing it now. I think i mean `pubObj[JS(meta)].as_object()` that you are looking up a second time here